x-pack/plugins/secops/server/graphql/sources/resolvers.test.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { GraphQLResolveInfo } from 'graphql';
+import { Sources, SourcesAdapter } from '../../lib/sources';
+import { createSourcesResolvers, SourcesResolversDeps } from './resolvers';
+import { mockSourceData } from './source.mock';
+
+const mockGetAll = jest.fn();
+mockGetAll.mockResolvedValue({
+  default: {
+    ...mockSourceData.configuration,
+  },
+});
+const mockSourcesAdapter: SourcesAdapter = {
+  getAll: mockGetAll,
+};
+const mockLibs: SourcesResolversDeps = {
+  sources: new Sources(mockSourcesAdapter),
+};
+const context: any = {
```

Comment:
can this `any` be narrowed to a specific type or interface?

Reply:
I will create an issue for it for later on